storage/innobase/xtrabackup/src/backup_mysql.cc

```diff
@@ -363,6 +364,25 @@ void parse_show_engine_innodb_status(MYSQL *connection) {
   mysql_free_result(mysql_result);
 }
 
+/* find the pxb base version */
+static unsigned long pxb_base_version() {
+  std::string pxb_base = MYSQL_SERVER_VERSION;
+  unsigned long major = 0, minor = 0, version = 0;
+  std::size_t major_p = pxb_base.find(".");
+  if (major_p != std::string::npos) major = stoi(pxb_base.substr(0, major_p));
+
+  std::size_t minor_p = pxb_base.find(".", major_p + 1);
+  if (minor_p != std::string::npos)
+    minor = stoi(pxb_base.substr(major_p + 1, minor_p - major_p));
+
+  std::size_t version_p = pxb_base.find(".", minor_p + 1);
+  if (version_p != std::string::npos)
+    version = stoi(pxb_base.substr(minor_p + 1, version_p - minor_p));
+  else
+    version = stoi(pxb_base.substr(minor_p + 1));
+  return major * 10000 + minor * 100 + version;
+}
+
```

Comment:
Does it work for debug binaries? And if the server is debug version?

Reply:
yes, it works for debug version,  release version and patch version. 
i.e it works for all version like 8.0.12 or  8.0.12.debug or 8.0.12.1